validator/db/kv/attestation_history_v2.go

```diff
@@ -45,7 +47,7 @@ func (store *Store) AttestationHistoryForPubKeyV2(ctx context.Context, publicKey
 		bucket := tx.Bucket(newHistoricAttestationsBucket)
 		enc := bucket.Get(publicKey[:])
 		if len(enc) == 0 {
-			attestationHistory = NewAttestationHistoryArray(0)
+			attestationHistory = NewAttestationHistoryArray(53999)
```

Comment:
this is a tweak for getting the full size array in interop on slot 1.
have to remove !!!!